src/ui/include/mainwindow.h

```diff
@@ -293,6 +293,7 @@ private slots:
     QString             currentWordOrSelection();
     void                currentWordOnlineSearch(const QString &searchUrl);
     QString             getNewDocumentName();
+    void                showSearchReplace(frmSearchReplace::Tabs tab);
```

Comment:
One last round of nagging and I'll be content.

Could you add a short documentation to this new function?